server.js

```diff
@@ -1,25 +1,24 @@
-const liveServer = require('live-server')
-const isSSR = !!process.env.SSR
-const middleware = []
+const liveServer = require('live-server');
+const isSSR = !!process.env.SSR;
+const middleware = [];
 
 if (isSSR) {
-  const Renderer = require('./packages/docsify-server-renderer/build.js')
+  const { initJSDOM } = require('./test/_helper');
+
+  const dom = initJSDOM();
+  dom.reconfigure({ url: 'https://127.0.0.1:3000' });
+
+  require = require('esm')(module /* , options */);
+
+  const {
+    Renderer,
+    getDefaultTemplate,
+  } = require('./packages/docsify-server-renderer/index');
+
+  debugger;
```

Comment:
temporary. I will delete this commit and recreate the branch all cleaned up. We just need to decide what to do.